src/lib/receiver.js

```diff
@@ -49,6 +56,7 @@ function createReceiver (opts) {
   const sspReceiverId = SSP_RECEIVER_ID_PREFIX + receiverId + '.'
   debug('receiver id: ' + receiverId)
 
+  const reviewPayment = opts.reviewPayment || (() => Promise.resolve(null))
```

Comment:
Agree with Evan - you should have to turn on SSP explicitly by providing a `reviewPayment` function.

I think it would be acceptable if `reviewPayment` was allowed to return either `null` or `Promise<null>`.

That way, I can do `reviewPayment: () => null` and just generally skip having to generate a Promise if I'm only going to do synchronous checks.